CoreScriptsRoot/StarterScript.lua

```diff
@@ -20,12 +20,15 @@ local luaControlsSuccess, luaControlsFlagValue = pcall(function() return setting
 local vrKeyboardSuccess, vrKeyboardFlagValue = pcall(function() return settings():GetFFlag("UseVRKeyboardInLua") end)
 local useVRKeyboard = (vrKeyboardSuccess and vrKeyboardFlagValue == true)
 
--- MainBotChatScript (the Lua part of Dialogs)
+-- MainBotChatScript (the Lua part of Dialogs)
 scriptContext:AddCoreScriptLocal("CoreScripts/MainBotChatScript2", RobloxGui)
 
 -- In-game notifications script
 scriptContext:AddCoreScriptLocal("CoreScripts/NotificationScript2", RobloxGui)
 
+-- Performance Stats Management
+scriptContext:AddCoreScriptLocal("CoreScripts/PerformanceStatsManagerScript", RobloxGui)
```

Comment:
this is a rather large change, so I'd like to flag it here, instead of inside the code just in case something happens from the time the flag is read until we do something about it